pkg/option/option.go

```diff
@@ -159,7 +158,7 @@ func New() *Options {
 
 	opt.flags.IntVar(&opt.StatusUpdateMaxBatchSize, "status-update-max-batch-size", 20, "Number of object statuses to update at maximum in one transaction.")
 
-	opt.viper.BindPFlags(opt.flags)
+	_ = opt.viper.BindPFlags(opt.flags)
```

Comment:
why make this change and the changes at L183, L187?

Reply:
this is a minor update because the IDE will display a warning if we don't handle the error.

Reply:
that's ok, but may I know which IDE you are using? On my side, vs code never reported this warning, and this could not be a help as the error is still not handled.

Reply:
I'm using IntelliJ IDEA, and here's an example where a warning is displayed in IntelliJ IDEA:
<img width="1086" alt="image" src="https://user-images.githubusercontent.com/16079222/218922898-3c03da15-e4e8-4522-be2f-d55038b4c20b.png">